src/System.Xml.Serialization/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,4 @@
+DEFAULT_APPNAME Error: 0 : Unable to resolve assembly 'Assembly(Name=mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)' referenced by the contract assembly 'Assembly(Name=System.Xml.Serialization, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089)'.
```

Comment:
I'm not sure why this is on the baseline, but it will go away once we change the facades to be written in code instead of using a depproj.